Detectors/TOF/workflow/src/TOFRawWriterSpec.cxx

```diff
@@ -86,6 +100,19 @@ void RawWriter::run(ProcessingContext& pc)
   encoder.flush();
   encoder.close();
 
+  // create configuration file for rawreader
+  std::ofstream fconfig;
+  fconfig.open("TOFraw.cfg");
+  fconfig << "[defaults]\n";
```

Comment:
This is what I meant in the my initial request to write the config: if the cfg file contains defaults for a particular detector, it makes impossible to concatenate the configs of different detectors. Would you mind to not write (or write commented by ``#``)?

Reply:
Ok, I didn't get this point. I fixed it